docs/user-manual/editor/engine-compatibility.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Engine Compatibility
+sidebar_position: 12
+---
+
+## Introduction
+
+The Editor supports two major release streams of the engine:
+
+- The latest release of engine v1.x.x (**Engine V1**)
+  - This provides support for WebGL1 and WebGL2, but does not receive any new features, and only critical bug fixes.
+- The latest release of engine v2.x.x (**Engine V2**)
+  - This has no support for WebGL1 and supports WebGL2 and WebGPU for rendering, and receives new features and fixes.
+
+You are free to switch between these at any time.
+
+### Switching projects between Engine V1 and V2
+
+This process will involve migration and updating of scripts given differences in our Engine API between Engine V1 and V2. All details regarding the engine changes can be found [here](/user-manual/engine/migrations)
+
+To initiate this navigate to the settings panel and click the `SWITCH TO ENGINE V2` button to convert your project to using Engine V2 (This will require a confirmation before converting). If you wish to switch back, there is a `SWITCH TO ENGINE V1` button in the same location for Project V2.
```

Comment:
```suggestion
To initiate this, navigate to the settings panel and click the `SWITCH TO ENGINE V2` button to convert your project to using Engine V2 (this will require a confirmation before converting). If you wish to switch back, there is a `SWITCH TO ENGINE V1` button in the same location for a project on V2.
```